### PR TITLE
BCDA-3693: Add synthetic data for NG and CEC ACOs for local testing

### PR DESCRIPTION
**Pull request by msnook**
### Fixes [BCDA-3693](https://jira.cms.gov/browse/BCDA-3693)

We are progressing towards supporting NextGen and CEC ACOs (in addition to SSP ACOs).  To ensure we protect ourselves against regression, we should introduce synthetic data for NextGen and CEC ACOs, to be used in local CI tests.

### Change Details

- Added synthetic data for V994, V996, E9994, and E9996 (similar to how our existing SSP synthetic data is generated)
- Tied the synthetic data ingestion into the CI
- Added `TODO` statements, to be completed as part of BCDA-3692, for using the data and credentials in smoke tests

### Security Implications

- [ ] new software dependencies
- [ ] security controls or supporting software altered
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications
- [x] no PHI/PII is affected by this change

### Acceptance Validation

All tests pass locally and in Travis.  The integration of post-deployment smoke testing will be done in a separate `bcda-ops` PR.

### Feedback Requested

Please review.

### CODE REVIEW

**Review by PatKelsh — APPROVED** [review]
Looks good

File: Makefile [hunk 1]
```diff
@@ -93,20 +94,28 @@ load-synthetic-cclf-data:
 	docker-compose up -d api
 	docker-compose up -d db
 	# The "test" environment provides baseline CCLF ingestion for ACO
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev --environment=test'
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-auth --environment=test'
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=small --environment=test'
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=medium --environment=test'
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=large --environment=test'
-	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=extra-large --environment=test'
+	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev --environment=test'
```
Comment (mtrang1263) [inline Makefile:97]: Is it possible to loop over an array of acoSizes + environments?

File: Makefile [hunk 2]
```diff
@@ -94,28 +94,14 @@ load-synthetic-cclf-data:
...
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-auth --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-cec --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-cec-auth --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-ng --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev-ng-auth --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=small --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=medium --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=large --environment=test'
-	docker-compose run --rm api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=extra-large --environment=test'
+	for acoSize in dev dev-auth dev-cec dev-cec-auth dev-ng dev-ng-auth small medium large extra-large ; do \
```
Comment (PatKelsh) [inline Makefile:97]: This looks a lot neater

**Review by PatKelsh — APPROVED** [review]

**Review by mtrang1263 — APPROVED** [review]
LGTM!

**Pull request merged**